### PR TITLE
Add successful order page

### LINKED ISSUE #12: Create a thank you page that gets called with the order id after the api call is succesfull
Child of #3

### PR DESCRIPTION
**Pull request by Barraza44**
Closes #12

### CODE REVIEW

**Comment by valincius** [comment]
Now that the PR for placing an order is done, you can update that action so that it redirects to this page

File: src/Congo.RazorPages/Pages/OrderSuccessful.cshtml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@page
+@model OrderSuccessful
+
+@{
+    ViewData["Title"] = "Order Successful";
+}
+
+<div class="container-fluid d-flex vh-100 flex-column justify-content-center align-items-center">
+    <h1 class="text-center">Your order has been successfully placed!</h1>
+    <h2 class="h4 text-center text-muted">Order ID: @Model.OrderId </h2>
```
Comment (valincius) [inline src/Congo.RazorPages/Pages/OrderSuccessful.cshtml]: Order Id doesn't show up correctly here, I just see a bunch of zeroes

File: src/Congo.RazorPages/Pages/OrderSuccessful.cshtml.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+using System;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace Congo.RazorPages.Pages
+{
+    public class OrderSuccessful : PageModel
+    {
+        [FromQuery(Name = "orderId")] public Guid OrderId { get; set; }
```
Comment (valincius) [inline src/Congo.RazorPages/Pages/OrderSuccessful.cshtml.cs]:
```suggestion
        [FromQuery] public Guid OrderId { get; set; }
```

**Comment by valincius** [comment]
When I click "Go Back" after checking out, I can still see the "Thanks for your purchase" message on the product list. Can you remove that?

**Comment by Barraza44** [comment]
> 
> 
> When I click "Go Back" after checking out, I can still see the "Thanks for your purchase" message on the product list. Can you remove that?

How? I don't see anything 🤔

**Comment by valincius** [comment]
> > When I click "Go Back" after checking out, I can still see the "Thanks for your purchase" message on the product list. Can you remove that?
> 
> How? I don't see anything 🤔


Idk why this is 3fps but this should show it

**Review by valincius — APPROVED** [review]

**Pull request merged**